packages/cli/src/services/util.ts

```diff
@@ -1,6 +1,14 @@
 import * as path from 'path'
 import * as fs from 'fs/promises'
 import { Service } from 'ts-node'
+import * as gitRepoInfo from 'git-repo-info'
+
+export interface GitInformation {
+  commitId: string
```

Comment:
So we don't want the originUrl for now?

Reply:
@umutuzgur , IMO, we can move on with the `repoUrl` from the project config. Then, we could adjust it to fetch the origin URL and sanitize it.

Reply:
Also, the only way to gather the origin URL, is to run a child process and execute the git config .... I found some NPM packages, but they are doing this under the hood.

Reply:
We shouldn't have to. It should be in the config file at `.git/config`